keras_trainer/regularizations.py

```diff
@@ -0,0 +1,34 @@
+from keras.models import model_from_json
+from keras.layers import SeparableConv2D, Conv2D, Dense, BatchNormalization
+
+
+def set_model_regularization(model, regularization_function, bias_term=False):
+    """
+    Go through every Convolutional, Dense and BN layer in the model and apply the specified regularization function
+    Args:
+        model: Keras Machine Learning model
+        regularization_function: Regularization function to include in layers
+        bias_term: If true will add the regularization in the layer bias regularizer
+
+    Returns: Model with regularization function applied
+
+    """
+    # Save weights in case it is a pre-trained model to not lose them afterwards
+    model.save_weights("tmp.h5")
+
+    for layer_index, layer in enumerate(model.layers):
```

Comment:
Be nice if could select specific layers to add regularisation too

Reply:
Ok added